source/Xunit.Gherkin.Quick/CoreModel/Feature/FeatureClassInfo.cs

```diff
@@ -1,17 +1,28 @@
 ﻿using System;
 using System.Reflection;
+using System.Text.RegularExpressions;
 
 namespace Xunit.Gherkin.Quick
 {
     internal sealed class FeatureClassInfo
     {
         public string FeatureFilePath { get; }
+        private Regex _matcher;
+        public bool IsPattern { get { return FeatureFilePath.Contains("*"); } }
 
         private FeatureClassInfo(string featureFilePath)
         {
             FeatureFilePath = !string.IsNullOrWhiteSpace(featureFilePath)
                 ? featureFilePath
                 : throw new ArgumentNullException(nameof(featureFilePath));
+
+            if (this.IsPattern) {

```

Comment:
Since you will change the above IsPattern getter, here, you can check `featureFilePath.Contains("*")`, which is better here because it will only be done once.

Reply:
Good idea! Thanks!